platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java

```diff
@@ -1334,11 +1331,6 @@ public void init(
 
         // initialize data structures used for FCQueue transaction records expiration
         initializeExpirationQueueAndAccountsSet();
-
-        logger.info(LOGM_DEMO_INFO, "Dual state received in init function {}", () -> new ApplicationDualStatePayload(
```

Comment:
i see the platform state was only being used here for logging purposes but do we not need this log line anymore? why?

Reply:
The only reason for the `platformState` parameter to exist in this method across all the implementation was this particular line of code. It used to be relevant but given the fact that it mentions the dual state, it's no longer the case.